plane/plane-tests/tests/backend_actions.rs

```diff
@@ -6,17 +6,18 @@ use plane::{
     database::backend::BackendActionMessage,
     names::{DroneName, Name},
     protocol::{BackendAction, Heartbeat, MessageFromDrone, MessageToDrone},
-    types::{ConnectRequest, ConnectResponse, ExecutorConfig, SpawnConfig},
+    types::{ClusterName, ConnectRequest, ConnectResponse, ExecutorConfig, SpawnConfig},
 };
 use plane_test_macro::plane_test;
 use std::time::Duration;
 
 mod common;
 
 /// Return a dummy connect request, which does not use a key.
-fn connect_request() -> ConnectRequest {
+fn connect_request(cluster: &ClusterName) -> ConnectRequest {
     ConnectRequest {
         spawn_config: Some(SpawnConfig {
+            cluster: Some(cluster.clone()),
```

Comment:
Spawn config now takes an optional cluster name. If it is not provided, we fall back on the default cluster name set on the controller.